zh_cn.lang

```diff
@@ -0,0 +1,457 @@
+gui.xaero_terrain_depth=地形深度
+gui.xaero_terrain_slopes=地形坡度
+gui.xaero_block_transparency=方块透明度
+gui.xaero_team=队伍
+gui.xaero_world_server=世界/服务器
+gui.xaero_subworld_dimension=子世界/位面
+gui.xaero_switch_waypoint_set=切换路径点集合
+gui.xaero_waypoint_opacity_ingame=游戏内路径点不透明度
+gui.xaero_waypoint_opacity_map=地图上路径点透明度
+gui.xaero_hide_world_names=隐藏IP/世界名称
+gui.xaero_open_slime=按下按键来打开
+gui.xaero_options=选项
+gui.xaero_make_automatic=使世界/服务器成为自动
+gui.xaero_make_automatic_msg1=你确定吗？
+gui.xaero_make_automatic_msg2=这将在选定的世界/服务器和自动识别的世界/服务器之间交换所有的路径点数据，从而模拟自动获取路径点。修改前请确保你知道你在做什么。
+gui.xaero_make_multi_automatic=使子世界成为自动
+gui.xaero_make_multi_automatic_msg1=你确定吗？
+gui.xaero_make_multi_automatic_msg2=这将在选定的子世界和自动识别的子世界之间交换所有的路径点数据，从而模拟自动获取子世界路径点。修改前请确保你知道你在做什么。
+gui.xaero_delete_world=删除世界/服务器
+gui.xaero_delete_world_msg1=你要删除所选世界/服务器内的所有地图数据吗？
+gui.xaero_delete_world_msg2=此操作无法撤销！请三思而后行！
+gui.xaero_delete_multi_world=删除子世界
+gui.xaero_delete_multi_world_msg1=你要删除所选子世界内的所有地图数据吗？
+gui.xaero_delete_multi_world_msg2=此操作无法撤销！请三思而后行！
+gui.xaero_always_show_distance=总是显示路径点的距离信息
+gui.xaero_item_tooltip=物品工具提示
+gui.xaero_entity_tooltip_settings=物品工具提示设置
+gui.xaero_item_tooltip_min_lines=最小行数
+gui.xaero_item_tooltip_time=显示格式
+gui.xaero_multiply_all_by_8=子世界坐标 * 8
+gui.xaero_divide_all_by_8=子世界坐标 / 8
+gui.xaero_multiply_msg1=此选项只用于修正不正确的坐标系
+gui.xaero_multiply_msg2=你确定你想要将§c§l所有§r§f子世界的坐标乘以 8 吗？
+gui.xaero_divide_msg2=你确定你想要将§c§l所有§r§f子世界的坐标除以 8 吗？
+gui.xaero_share=广播坐标
+gui.xaero_share_msg1=你确定要将这个路径点广播给聊天频道中的§c所有人§f吗？
+gui.xaero_share_msg2=请确保这样做不会暴露你的秘密坐标！
+gui.xaero_roof=方块
+gui.xaero_partial=部分的
+gui.xaero_full=全部的
+gui.xaero_show_light_level=显示光线强度等级
+gui.xaero_up=[向上]
+gui.xaero_down=[向下]
+gui.xaero_display_all_sets=所有路径点渲染开关
+gui.xaero_waypoint_name=路径点名称
+gui.xaero_set_name=集合名称
+gui.xaero_render_layer=覆盖渲染层
+gui.xaero_visit_needed=§e你需要首先来到这个位面，才能将它转换为新的格式！
+gui.xaero_mp_cross_tp=多人模式跨位面传送
+gui.xaero_cross_tp=跨位面传送
+gui.xaero_unreachable_dimension=目标位面不可达
+gui.xaero_show_time=显示游戏时间
+gui.xaero_24h=24 小时制
+gui.xaero_12h=12 小时制
+gui.xaero_temporary=临时的
+gui.xaero_restore=还原
+gui.xaero_biomes_vanilla=以原版颜色显示生物群系
+gui.xaero_potion_effects_blink=当药水效果准备消失时闪烁
+gui.xaero_use_multiworld=多世界检测
+gui.xaero_close=关闭
+gui.xaero_waypoint_distance_visibility_angle=航向角阈值
+gui.xaero_waypoint_distance_vertical_visibility_angle=俯仰角阈值
+gui.xaero_teleportation=传送
+gui.xaero_teleportation_tooltip=禁用后可以在配置文件中重新启用
+gui.xaero_use_multiworld_tooltip=如果这是一个单世界的简单服务器（没有登录大厅、不同的游戏模式、或者小游戏世界），请禁用此项。在这样的服务器上多世界检测会导致问题。不过在服务器上安装此模组可以避免前述问题。
+gui.xaero_world_teleport_command=传送命令
+gui.xaero_use_default=使用默认值
+gui.xaero_centered_enlarged=展开时居中
+gui.xaero_zoomed_out_enlarged=展开时拉远
+gui.xaero_sort=排序
+gui.xaero_sort_reversed=倒序
+gui.xaero_sort_unsorted=不排序
+gui.xaero_sort_name=名称
+gui.xaero_sort_symbol=首字母
+gui.xaero_sort_distance=距离
+gui.xaero_box_cave_maps=一个可以显示地面之下的地图模式，主要用来显示地底的洞穴和建筑内部。\n 该参数指的是在你的头顶上的“正方形”范围内有固实方块遮挡时启用洞穴模式。
+gui.xaero_box_entity_amount=指定每个实体类别的容量极限。
```

Comment:
Here the original file is "The limit per each entity category."
So for example, if I have cows more than this limit, those cows that "overflow" will not display on minimap, right?

Reply:
It refers to the entity categories like friendly mobs, hostile mobs, players. So cows and chickens would go in 1 category and they are limited separately from hostile mobs etc.